lib/interpreters/split_clustering.rb

```diff
@@ -481,6 +481,9 @@ def self.remove_poorly_populated_routes(vrp, result, limit)
 
         if load_flag && time_flag
           emptied_routes = true
+          %i[total_distance total_time total_travel_time total_value].each{ |key|
+            result[key] -= route[key] if route[key]
+          }
```

Comment:
Good catch :+1:  

This remainds me the internal mode::result class will need it own add/update/remove/empty functions to handle the logic inside the result class so that split_clustering (for example) does not have to have a result related update logic. It looks like it is going to be you whose going to implement the result model, can you make a note of this comment. You can just copy/paste the comment link if there is a ticket.